cedar-drt/initialize_corpus.sh

```diff
@@ -47,6 +47,11 @@ initialize_corpus() {
       cp "$f" "$corpus_dir/$(uuidgen)-$(basename "$f")"
     done ;;
 
+  convert-policy-json-to-cedar)
```

Comment:
Not the fault of this PR, but I noticed a typo "arguemtn" in this file on L36 - can you fix in this PR?